kubernetes/launch-job.sh

```diff
@@ -53,14 +66,29 @@ usage() {
   echo "    /shared/victimplay/ttseng-avoid-pass-alive-coldstart-39-20221025-175949/models/t0-s545065216-d136760487"
   echo "    for the cyclic-adversary. The corresponding model file is expected"
   echo "    to be in either the TF weights directory or /shared/victims/."
+  echo "  --victim-ckpt CHECKPOINT"
+  echo "    Only for and is required for iterated training. Path to the TF weights"
+  echo "    directory for the last victim in the initial curriculum."
```

Comment:
Am a little confused about this naming due to the victim actually alternating between the cyclic policy and KataGo. Which victim is this referring to?